lib/helper/WebDriverIO.js

```diff
@@ -1702,4 +1712,18 @@ function isFrameLocator(locator) {
   return locator[key];
 }
 
+function prepareLocateFn(context) {
+  if (!context) return this._locate.bind(this);
+  let el;
```

Comment:
Sure, el is defined few lines below:

```js
return this.browser.elementIdElements(el = res.value[0].ELEMENT, l);
```
This is done to cache the context element and reuse it if it was already found